architecture/sync.md

```diff
@@ -296,10 +277,7 @@ The steps to resolve keystore conflicts are as follows:
 
 #### Item Conflicts
 
-Reconciling conflicting item changes start with the following basic rules:
-
-* _Favor "update" actions over "remove" actions_. If an item is both removed and updated, treat it as updated.
-* _Favor local changes over remote changes_. For a given change within an item, favor the local change over the remote change.
+Reconciling conflicting changes within an item adheres to the basic theme **_favor `local` changes over `remote` changes_**.  The rationale is that any changes made on a device trigger an immediate sync, and therefore are more likely to match the user's current intent.
 
 Item conflicts are reconciled as follows:
 
```

Comment:
Uber-nit: Bulleted list below is funny in that it starts at "2." (in both the Markdown and rendered output).

Reply:
My jekyll render (which is what's used to get https://mozilla-lockbox.github.io/architecture/sync.html) looks correct to me; maybe it's something with Github's auto-render?

I can correct the numbering in the text, though.  That's a hold-over from a slew of edits.